src/widgets/viewarea.cpp

```diff
@@ -430,6 +431,14 @@ ViewWindow *ViewArea::getCurrentViewWindow() const
     return nullptr;
 }
 
+int ViewArea::getCurrentViewWindowIndex()
+{
+    auto split = getCurrentViewSplit();
+    if (split) {
+        return split->getCurrentViewWindowIndex();
+    }
```

Comment:
No return value if `split` is `nullptr`?

Anyway, this function is unnecessary. ViewSplit knows its current window index, so ViewArea does not need to tell him.

Reply:
done